src/tests/screens/Home/surveyList.test.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+
+import { render } from '@testing-library/react'
+
+import SurveyList from 'screens/Home/surveyList'
+
+describe('given surveyList page is mounted', () => {
+  describe('given valid survey list', () => {
+    it('renders two surveys', () => {
+      const surveyList = [
+        {
+          node: {
+            id: '1',
+            title: 'title-1',
+            description: 'description-1',
+            coverImageUrl: 'image-1'
+          }
+        },
+        {
+          node: {
+            id: '2',
+            title: 'title-2',
+            description: 'description-2',
+            coverImageUrl: 'image-2'
+          }
+        }
+      ]
+
+      const { container } = render(<SurveyList surveyList={surveyList} />)
+      const surveys = container.getElementsByClassName('survey-list-data')
+
+      expect(surveys.length).toBe(2)
+    })
+
+    it('renders correct survey title', () => {
```

Comment:
How about taking this section in a new describe block? - 
```
describe('given a survey in the survey list', () => {
  it('renders the correct survey title', () => {
  it('renders correct survey description', () => {
  it('renders correct survey image', () => {
```

Reply:
Could you please explain more why you think we need to move to the new describe block? I think the current implementation makes sesne.